docs/geedocs/5.2.4/index.html

```diff
@@ -1,47 +1,25 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-<title>Google Earth Enterprise Documentation 5.2.4</title>
-<link rel="stylesheet" href="css/styles.css" type="text/css" media="all" />
-<link rel="stylesheet" href="css/containers.css" type="text/css" media="all" />
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-108632131-2"></script>
-<script>
-window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
-gtag('js', new Date());
-gtag('config', 'UA-108632131-2');
-</script>
-</head>
+---
```

Comment:
I'm running into a problem here. This works great for the website, but everything under the 5.2.4 folder is installed with GEE Server or Fusion locally as static html files so that help is available offline. In that case these files are not processed by Jekyll, they're treated as raw HTML files, so this doesn't work. To see what I mean, you can build and install Fusion, open the Fusion UI, and go to Help -> Contents. You get something like this:
![image](https://user-images.githubusercontent.com/9662346/43849368-1b3108c8-9aea-11e8-9c02-734db98ff784.png)
Is there another way to include the analytics code that will work for pure HTML?

Reply:
@tst-lsavoie I will see if there's a way to automate adding the Google analytics tag to GEE Server- or Fusion-installed html files. I'm sure there must be a way to do this so that the tags don't have to be added manually for each new documentation version.

Reply:
@tst-lsavoie Didn't you mention that the build could run the files through Jekyll at some point?